libpod/image/image.go

```diff
@@ -916,6 +947,68 @@ func splitString(input string) string {
 	return split[len(split)-1]
 }
 
+// IsParent goes through the layers in the store and checks if i.TopLayer is
+// the parent of any other layer in store. Double check that image with that
+// layer exists as well.
+func (i *Image) IsParent() (bool, error) {
+	children, err := i.GetChildren()
+	if err != nil {
+		return false, err
+	}
+	return len(children) > 0, nil
+}
+
+// GetParent returns the image ID of the parent
```

Comment:
Should mention that this returns nil if there is no parent

Reply:
Fixed